buildspec.yml

```diff
@@ -5,15 +5,17 @@ version: 0.2
 phases:
   install:
     runtime-versions:
-      nodejs: 14
-      ruby: 2.7
+      nodejs: 16
+      ruby: 3,1
```

Comment:
I wonder why this says 3,1? You mean 3.0?

Reply:
that comma was a typo (fixed now), however, the runtime from CodeBuild only includes Ruby 3.1, so that's what we have to ask for.